src/wireviz/Harness.py

```diff
@@ -110,14 +110,24 @@ def create_graph(self) -> Graph:
                 pinhtml = []
                 pinhtml.append('<table border="0" cellspacing="0" cellpadding="3" cellborder="1">')
 
-                for pin, pinlabel in zip(connector.pins, connector.pinlabels):
+                for pin, pinlabel, pincolor in zip(connector.pins,
+                                                   connector.pinlabels,
+                                                   connector.pincolors if connector.pincolors else [None] * len(connector.pins)):
```

Comment:
I looked at options such as [using `itertools.zip_longest`](https://stackoverflow.com/questions/1277278/is-there-a-zip-like-function-that-pads-to-longest-length-in-python) to simplify this, and not require the padding in `DataClasses.py` line 127; however, since we are already autofilling lists for wires' `colors` I deemed this way to be more consistent.